org.sofproject.gst.topo/src/org/sofproject/gst/topo/model/GstTopoNode.java

```diff
@@ -100,8 +100,8 @@ public GstTopoConnection getOutgoingConnection() {
 	public void serialize(Writer writer) throws IOException {
 		writer.write(elem.getName());
 		for (GstProperty prop : properties.values()) {
-			if (!prop.isChanged())
-				continue;
+			//if (!prop.isChanged())
```

Comment:
Do not leave the dead code unless commented why it is kept here (e.g. some TODO).  (There might be existing bad examples in the code that require cleanup).